open_humans/views.py

```diff
@@ -287,14 +288,20 @@ def get_context_data(self, *args, **kwargs):
         # is done later in the template.
         # TODO: Will be overhauled with Issue #809 to make this less
         # convoluted
+
         projects = DataRequestProject.objects.filter(
-            approved=True).filter(
-                active=True)
-        activities = sort_activities(
-            {proj.id_label: activity_from_data_request_project(
-                project=proj, user=self.request.user) for proj in projects})
+            approved=True).filter(active=True).exclude(
+                returned_data_description__isnull=True).exclude(
+                    returned_data_description__exact='')
+        if not self.request.user.is_anonymous:
+            project_memberships = DataRequestProjectMember.objects.filter(
```

Comment:
When I test locally it seems that this filter isn't working - I log in with my user and see projects listed that I've joined...

Also, previously this list was sorted according to number of members - I think we should keep doing that :)

Reply:
Also: ExploreSharePageView was re-using this view with another template.

But because the filtering on "data source" was moved from template to view (probably appropriate), the ExploreSharePageView is now broken -- it's producing the same list.